Content.Server/Radio/EntitySystems/JammerSystem.cs

```diff
@@ -78,11 +84,47 @@ private void OnRadioSendAttempt(ref RadioSendAttemptEvent args)
         var query = EntityQueryEnumerator<ActiveRadioJammerComponent, RadioJammerComponent, TransformComponent>();
         while (query.MoveNext(out _, out _, out var jam, out var transform))
         {
-            if (source.InRange(EntityManager, _transform, transform.Coordinates, jam.Range))
+            if (source.InRange(EntityManager, _transform, transform.Coordinates, GetCurrentRange(jam)))
             {
                 args.Cancelled = true;
                 return;
             }
         }
     }
+
+    private void OnGetVerb(Entity<RadioJammerComponent> entity, ref GetVerbsEvent<Verb> args)
+    {
+        if (!args.CanAccess || !args.CanInteract)
+            return;
+
+        var user = args.User;
+
+        for (byte i = 0; i < entity.Comp.Settings.Count; i++)
```

Comment:
Could this be 
```csharp
foreach(RadioJamSetting in entity.Comp.Settings) 
```

Reply:
the index is needed for the verb action, and var would be used in that case

Reply:
Yes it could! See this post that I made like RIGHT before you posted that :laughing:

https://github.com/space-wizards/space-station-14/pull/25546#discussion_r1501892568

Basically I could do that but I still need to save the index so it gets a little weird...
Let me know what you think I should do!